bigtable/api/README.md

```diff
@@ -0,0 +1,204 @@
+# .NET Cloud Bigtable Samples
+
+A collection of samples that demonstrate how to call the
+[Google Cloud Bigtable API](https://cloud.google.com/bigtable/docs/) from C#.
+
+This sample requires [.NET Core 2.0](https://www.microsoft.com/net/core) or later.  That means using [Visual Studio 2017](https://www.visualstudio.com/), or the command line.
+
+## Build and Run
+
+1.  **Follow the instructions in the [root README](../../README.md)**.
+
+2.  Enable APIs for your project.
+    [Click here](https://console.cloud.google.com/flows/enableapi?apiid=bigtable&showconfirmation=true)
+    to visit Cloud Platform Console and enable the Google Cloud Bigtable API.
+
+## Quick Start
+
+3.  Provisioning an instance 
+    Follow the instructions in the [user
+    documentation](https://cloud.google.com/bigtable/docs/creating-instance) to
+    create a Cloud Bigtable instance if necessary.
+
+4.  Follow the [cbt tutorial](https://cloud.google.com/bigtable/docs/quickstart-cbt) to install the cbt       command line tool. Here are the cbt commands to create a table, column       family and add some data:
+    ```
+    cbt -project <YOUR-PROJECT-ID> -instance<YOUR-INSTANCE-ID> createtable my-table
```

Comment:
Update add a space between instance and <YOUR-INSTANCE-ID> 
Should be: "-instance <YOUR-INSTANCE-ID>"

Also update the next two lines in the same way.

Reply:
[Comment got truncated...]

Add a space between "instance" and &lt;YOUR-INSTANCE-ID>